addons/website_slides/controllers/main.py

```diff
@@ -831,12 +898,12 @@ def create_slide(self, *args, **post):
 
         redirect_url = "/slides/slide/%s" % (slide.id)
         if channel.channel_type == "training" and not slide.slide_type == "webpage":
+            redirect_url = "/slides/slide/%s" % (slide.id)
+            # redirect_url = "/slides/%s" % (slug(channel))
```

Comment:
remove this. also, this test makes no sense as redirect url is already defined with exact same value just before.